### ISSUE TITLE
Fanart

### ISSUE THREAD

**Issue by r95sguy** [mintsoft/kodi.plugin.ytchannels#44]
can an option to use video thumbnails as fanart be added? Fanart is used as background in different Kodi skins. Thanks!

**Comment by mintsoft** [comment]
Yeah that seems possible, whether there is a high-enough quality image to be used as fanart would be the only real issue I can think of

**Comment by r95sguy** [comment]
Thank you for looking into this! having this option would be great in cases of a good quality thumb image, it can be disabled in the other cases.
